includes/admin/post-types/meta-boxes/class.llms.meta.box.course.options.php

```diff
@@ -356,7 +357,18 @@ public function get_fields() {
 			),
 		);
 
-		if ( function_exists( 'register_block_type' ) && llms_blocks_is_post_migrated( $this->post->ID ) ) {
+		$current_screen = get_current_screen();
+		$is_gutenberg   = method_exists( $current_screen, 'is_block_editor' ) && $current_screen->is_block_editor();
```

Comment:
This is a bit paranoid I know but what about
```suggestion
		$is_gutenberg   = is_object( $current_screen ) && method_exists( $current_screen, 'is_block_editor' ) && $current_screen->is_block_editor();
```

Reply:
you never know, prevention is always better than cure :D 

thanks, all done 👍